src/audio/component.c

```diff
@@ -234,3 +234,33 @@ void audio_stream_copy_to_linear(const struct audio_stream __sparse_cache *sourc
 		snk += bytes_copied;
 	}
 }
+
```

Comment:
@btian1 why will cmocka tests require comp_copy() On the contrary the comp_drv interface will eventually be deprecated for all modules.

Reply:
> 

Yes, Marc, this is correct, regarding the #ifndef __ZEPHYR__, I will change it in #6322(since perf_cnt.h need change as well), after this merged, #6322 should be based on ipc4+zephyr, ipc4 need this change as must, so I have to wait for this change merge first, then modify #6322.